tensorflow_addons/image/BUILD

```diff
@@ -1,3 +1,49 @@
 licenses(["notice"])  # Apache 2.0
 
 package(default_visibility = ["//visibility:public"])
+
+
+cc_binary(
+    name = "python/_image_ops.so",
+    srcs = [
+        "cc/kernels/image_projective_transform_op.cc",
+        "cc/kernels/image_projective_transform_op.h",
+        "cc/ops/image_ops.cc",
+    ],
+    linkshared = 1,
+    deps = [
+        "@local_config_tf//:libtensorflow_framework",
+        "@local_config_tf//:tf_header_lib",
+    ],
+    copts = ["-pthread", "-std=c++11", "-D_GLIBCXX_USE_CXX11_ABI=0"]
```

Comment:
Just something to note for the future... After speaking with SIG-Build I think we're going to restructure our build so all copts exist in one central location.

Reply:
Agreed!